crates/libs/core/src/client/svc_mgmt_client.rs

```diff
@@ -87,6 +108,20 @@ impl ServiceManagementClient {
         let res = ResolvedServicePartition::from_com(com);
         Ok(res)
     }
+
+    /// Simulates a service replica failure by restarting a persisted service replica,
+    /// closing the replica, and then reopening it. Use this to test your service for problems
+    /// along the replica reopen path. This helps simulate the report fault temporary path through client APIs.
+    /// This is only valid for replicas that belong to stateful persisted services.
+    pub async fn restart_replica(
```

Comment:
should we name it as restart_stateful_replica?

Reply:
I prefer keep the same name as the COM api.